LEGO1/omni/src/video/mxdisplaysurface.cpp

```diff
@@ -434,7 +434,7 @@ void MxDisplaySurface::VTable0x28(
 	MxBITMAPINFO* bmi = p_bitmap->GetBitmapInfo();
 	if (bmi) {
 		PALETTEENTRY pe[256];
-		for (int i = 0; i < 256; i++) {
+		for (int i = 1; i < 256; i++) {
```

Comment:
What does this change do?

Reply:
Makes the edges of scaled transparent surfaces black instead of purple, it only affects devices that have resolutions less then 640x480